components/ontology-access/hgnc/api/src/main/java/org/phenotips/ontology/internal/GeneNomenclature.java

```diff
@@ -151,7 +151,8 @@ public OntologyTerm getTerm(String id)
     {
         OntologyTerm result = this.cache.get(id);
         if (result == null) {
-            HttpGet method = new HttpGet(fetchServiceURL + "symbol/" + id);
+            id = id.replaceAll("\\s", "");
```

Comment:
Are you sure it's OK to just remove whitespace? You should encode the ID instead.